pro/src/components/IndividualOfferForm/utils/setDefaultInitialFormValues.ts

```diff
@@ -41,13 +44,19 @@ const setDefaultInitialFormValues = (
     }
   }
 
+  const { subcategoryFields } = buildSubcategoryFields(
+    isBookingContactEnabled,
+    subcategory
+  )
+
   return {
     ...FORM_DEFAULT_VALUES,
     isDuo: subcategory?.canBeDuo ?? FORM_DEFAULT_VALUES.isDuo,
     categoryId: subcategory?.categoryId ?? FORM_DEFAULT_VALUES.categoryId,
     subcategoryId: subcategory?.id ?? FORM_DEFAULT_VALUES.subcategoryId,
-    subCategoryFields:
-      subcategory?.conditionalFields ?? FORM_DEFAULT_VALUES.subCategoryFields,
+    subCategoryFields: subcategoryFields
```

Comment:
On peut garder le même comportement ?
```suggestion
    subCategoryFields: subcategoryFields ?? FORM_DEFAULT_VALUES.subCategoryFields,
```

Reply:
nope parce que `subcategoryFields` est pas null / undefined quand il est vide donc ça utiliserait pas FORM_DEFAULT_VALUES.subCategoryFields